docs/features/config/replace-configuration-tokens-with-isecretprovider.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Replace configuration tokens with ISecretProvider"
+layout: default
+---
+
+The `Arcus.WebApi.Security` package provides a mechanism to use your own `ISecretProvider` implementation when building your configuration for your web application.
+
+### Usage
+
+When building your `IConfiguration`, you can call the extension `.AddAzureKeyVault` to pass in your version of the `ISecretProvider` instead on using a specific key vault store name of client.
```

Comment:
> instead of on using a specific key vault store name of client

What do you mean with this?

Reply:
Those are the default `.AddAzureKeyVault` overloads.

Reply:
I know, but we should be a bit more descriptive about it.

For example:
> When building your `IConfiguration`, you can use the extension `.AddAzureKeyVault` to pass in your `ISecretProvider` instead of using the built-in [Azure Key Vault provider](https://docs.microsoft.com/en-us/aspnet/core/security/key-vault-configuration?view=aspnetcore-2.2#packages).

Reply:
👍

Reply:
Thanks!